runtime/vm/JVMImage.cpp

```diff
@@ -632,18 +635,23 @@ extern "C" void
 registerClass(J9JavaVM *javaVM, J9Class *clazz)
 {
 	IMAGE_ACCESS_FROM_JAVAVM(javaVM);
-
 	Assert_VM_notNull(jvmImage);
 
+	J9ROMClass *romClass = clazz->romClass;
+
+	J9UTF8 *className = J9ROMCLASS_CLASSNAME(romClass);
+	Trc_VM_ImageClassRegister(javaVM->mainThread, clazz, J9UTF8_DATA(className));
```

Comment:
These tracepoints are causing issues in the build. I am currently investigating but not sure why. Reproduce steps:
1. Run save state with tracepoint on. `java -Xint -Xshareclasses -Xsaveramstate=test -Xtrace:print=j9vm.609 -version`
2. Run restore state `java -Xint -Xshareclasses -Xrestoreramstate=test -version`
I get a sigsev on classHashEqualFn. It doesn't happen if -Xtrace is off forfirst step

Reply:
I might remove them as they are not too important. Just wanted them for testing purposes, but @tajila, @Ershany-IBM, @VinoshanT  any thoughts on the issue?